### DISCUSSION TITLE
Upload file location

### DISCUSSION THREAD

**Discussion by GuilhermeDeLucca** [LiveHelperChat/livehelperchat#1881] [General]
I'm wanting to implement a function that will perform some validations like reading uploaded files E.g Name, Tracking code...

However I am having difficulty using getimagesize due to what I believe is the missing file extension.

The URL I am consulated is the following:
<domain>/site_admin/file/downloadfile/28/83cb9d66a787c96d30aebee67c55cc29/(inline)/true

Any suggestion?

Note: Also, make a not very elegant setting due to RASA interpreting empty ints when the visitor uploads, but we'll talk about that another time :p

**Comment by remdex** [comment]
Please see this extension how it intercepts file uploads and uploads file instead of S3 buckets
https://github.com/LiveHelperChat/amazon-s3/blob/master/amazons3/bootstrap/bootstrap.php
